App.js

```diff
@@ -16,17 +16,26 @@ function SideBar() {
   return (
     <div className="NavBarCol">
       <col />
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-
+      <div classNAme="ButtonRow">
+        <button className="Icon1" />
+        <button className="Icon" />
+        <p>Home</p>
+        <button className="Icon" />
+        <p>Explore</p>
+        <button className="Icon" />
+        <p>Notifications</p>
+        <button className="Icon" />
+        <p>Messages</p>
+        <button className="Icon" />
+        <p>Bookmarks</p>
+        <button className="Icon" />
+        <p>Lists</p>
+        <button className="Icon" />
+        <p>Profile</p>
+        <button className="Icon" />
+        <p>More</p>
```

Comment:
I'm not suggesting you do this right away, but eventually you can see how you could create a common `Icon` component that takes in a name:

```jsx
function Icon({ name }) {
  return (
        <button className="Icon" />
        <p>{name}</p>
  )
}
```